tests/utils/StringTest.cpp

```diff
@@ -32,11 +31,27 @@ TEST_P(WriteInt2mmTest, WriteInt2mm)
     ASSERT_TRUE(ss.good()) << "The integer " << in << " was printed as '" << str << "' which was a bad string!";
 
     const int out = MM2INT(strtod(str.c_str(), nullptr));
-    ASSERT_EQ(in, out) << "The integer " << in << " was printed as '" << str << "' which was interpreted as " << out << " rather than " << in << "!";
+    ASSERT_EQ(in, out) << "The integer " << in << " was printed as '" << str << "' which was interpreted as " << out << " rather than "
+                       << in << "!";
```

Comment:
Why is the final integer on a separate line?

Reply:
don't know probably ran into the line length limit, will check